src/intervals/conversion_promotion.jl

```diff
@@ -19,3 +11,103 @@ promote_rule{T<:Real, S<:Real}(::Type{Interval{T}}, ::Type{S}) =
 
 promote_rule{T<:Real}(::Type{BigFloat}, ::Type{Interval{T}}) =
     Interval{promote_type(T, BigFloat)}
+
+
+## Conversion rules
+
+# Just specifying Interval defaults to Float64 interval
+# Remove this default so that it is necessary to explicitly specify?
+convert(::Type{Interval}, x::Real) = convert(Interval{Float64}, x)
```

Comment:
Using your example, for `x=1//3` my point is that `convert(Interval,x)` should default in `Interval{typeof(x)}(x)`, but it doesn't:
```julia
julia> x = 1//3
1//3

julia> convert(Interval, x)
[0.3333333333333333, 0.33333333333333337]
```